### DISCUSSION TITLE
Support for Remix

### DISCUSSION THREAD

**Discussion by szmarci** [ben-rogerson/twin.macro#623] [Q&A] [answered]
Any play on supporting [Remix](https://github.com/remix-run)?

I tried to hack it together following the Next.js tutorial, since they are very similar server side frameworks, but I could not make it work.

`import tw, { styled } from "twin.macro";`

Importing like this ends up in `styled` is undefined:

I guess it is some bundling issue?

**Comment by ben-rogerson** [comment, marked as answer]
I'd love to support Remix but it doesn't seem to have support for babel right now.
Babel needs to be able to run so it can change the tailwind classes into css object + swap `styled` out with an import from styled-components.

**Reply by szmarci** [reply]
Ah, I see. Thanks.

**Comment by sudokoi** [comment]
@ben-rogerson now that remix uses vite, is there a pathway available to support remix?
